### ISSUE TITLE
Occurrence marker and rename - occurrence not marked/replaced

### ISSUE THREAD

**Issue by LievenDeFoor** [omecra/de.omecra.oedt.updatesite#163]
Hi,

I found a bug where the 'rename' functionality failed to see and rename an occurrence.

![afbeelding](https://user-images.githubusercontent.com/8639525/233955703-a8a47cc0-88e2-4bcc-9097-750b0af663d1.png)
![afbeelding](https://user-images.githubusercontent.com/8639525/233955992-53e283fc-4e66-49f0-8445-0c2e7df9bec1.png)
![afbeelding](https://user-images.githubusercontent.com/8639525/233956029-c018fdea-9e13-4c4c-8524-96f6b3f02f9b.png)

In our code there happened to be a class variable with the same name, so this code kept compiling so we introduced a bug...

```
CLASS OEDTRenameIssue.OEDTRenameIssue:

    DEFINE PUBLIC PROPERTY ProcedureHandle AS HANDLE NO-UNDO
    GET.
    PRIVATE SET.

    CONSTRUCTOR PUBLIC OEDTRenameIssue(ProcedureHandleInput AS HANDLE):

        RUN Dummy IN ProcedureHandleInput("x":U).
        RUN Dummy2 IN ProcedureHandleInput.

        Dummy3(ProcedureHandleInput).

    END CONSTRUCTOR.

    METHOD PRIVATE VOID Dummy3(ProcedureHandleInput AS HANDLE):

        ASSIGN ProcedureHandle = ProcedureHandleInput.

    END METHOD.

END CLASS.
```

**Comment by LievenDeFoor** [comment]
It looks like the occurrence that isn't detected is seen as a method call.
If I create a new method "ProcedureHandleInput" then the occurrence marker marks it when selecting ProcedureHandleInput in the RUN Dummy statement...

**Comment by lneumeier** [comment]
Yes, that's the issue. I look into it.

**Comment by lneumeier** [comment]
Fixed with OEDT 9.4.8.
